tests/e2e/vmlistener_test.go

```diff
@@ -120,7 +124,7 @@ func (s *VMListenerTestParams) TestKillVM(c *C) {
 
 	// power on vm
 	govc.PowerOnVM(s.dockerHostName)
-	misc.SleepForSec(35)
+	misc.SleepForSec(45)
```

Comment:
I am not sure if going ahead with tests if VM state is desired one, especially power on is a good idea. there is still a delay where docker itself tries to come up, it itself tries to talk to plugin (btw it does exponential backoff) and then plugin comes up and stabilizes refcount. 
If this isn't clear, we can quickly discuss offline.